helm-charts/policy/templates/config_policy_service_account.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright Contributors to the Open Cluster Management project
+
+# Licensed Materials - Property of IBM
```

Comment:
Could you please remove these IBM headers from new files?

Reply:
Good idea